app/src/components/Buttons/SimpleButton.svelte

```diff
@@ -1,13 +1,15 @@
 <script lang="ts">
 	export let value: string = '';
 	export let disabled: boolean = false;
+
+	let className = '';
+	export { className as class };
+
+	$: classInside =
```

Comment:
If the classes are dynamic, this might make sense, but static classes don't need to be put in a separate variable